README.md

```diff
@@ -265,3 +265,9 @@ And restart apache
 sudo systemctl restart apache2
 ```
 
+
+### Change the remote repository URL
+```
+git remote set-url origin <new-repository-URL>
```

Comment:
`origin` can be anything else.

for example:

```
git remote set-url gitlab <new-repository-url>
```

I would be better to have it like this:
```suggestion
git remote set-url <remote-name> <new-repository-URL>

// example
git remote set-url origin https://github.com/shetabit/ubuntu-practical.git
```